app/controllers/departments_controller.rb

```diff
@@ -1,5 +1,6 @@
 class DepartmentsController < ApplicationController
   before_action :set_department, only: [:show, :edit, :update, :destroy]
```

Comment:
In general, you may be better off not using before actions to achieve setup like this. Before actions let you achieve common things that are not directly related to the business logic of the method. For example, you should be able to directly see all things that are relevant to the logic in the function. Something like login doesn't impact the logic in the controller method functions. However, setting up variables are different. This may lead to frustrating bugs. I would recommend against this especially since your setup here involves only a single variable being setup.